src/Microsoft.PowerShell.Commands.Utility/commands/utility/RemoveAliasCommand.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+using System.Management.Automation;
+using System.Management.Automation.Internal;
+
+namespace Microsoft.PowerShell.Commands
+{
+    /// <summary>
+    /// The implementation of the "Remove-Alias" cmdlet.
+    /// </summary>
+    ///
+    [Cmdlet(VerbsCommon.Remove, "Alias", DefaultParameterSetName = "Default", HelpUri = "")]
+    public class RemoveAliasCommand : PSCmdlet
```

Comment:
Please add an alias for Remove-Alias at : https://github.com/PowerShell/PowerShell/blob/master/src/System.Management.Automation/engine/InitialSessionState.cs#L4656
 I recommend `ral` so we are consistent with `gal`,`sal` and `nal`